iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -880,6 +885,11 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
                             return false; // Let anything else stop the application.
                         });
                     }
+                    catch (DotNetty.Transport.Channels.ConnectException ex)
+                    {
+                        //same as above, we will handle DotNetty.Transport.Channels.ConnectException
+                        Debug.WriteLine("ConnectException trying to connect to {0}: {1}", address.ToString(), ex.ToString(), nameof(CreateChannelFactory));
```

Comment:
Please follow logging guidelines (see the architecture.md file).

Reply:
(also this and other files)

Reply:
Sure, I'll fix this to use the proper logger like the above catch does